CHANGELOG/CHANGELOG-1.1.md

```diff
@@ -0,0 +1,94 @@
+- Add live-traffic Traceflow support to the Antrea [Octant] plugin, which includes support for displaying the captured packet's headers. ([#2124](https://github.com/antrea-io/antrea/pull/2124), [#2182](https://github.com/antrea-io/antrea/pull/2182), [@luolanzone])
+- Add crd.antrea.io/v1alpha3/ClusterGroup API resource which removes the deprecated "ipBlock" field; a [conversion webhook] is added to the Controller to convert from the v1alpha2 version to the v1alpha3 version. ([#2008](https://github.com/antrea-io/antrea/pull/2008), [@Dyanngg])
+- Add support for providing an IP address as the source for live-traffic Traceflow; the source can also be omitted altogether in which case any source can be a match. ([#2068](https://github.com/antrea-io/antrea/pull/2068), [@jianjuns])
+- Add ICMP echo ID and sequence number to the captured packet for live-traffic Traceflow. ([#2162](https://github.com/antrea-io/antrea/pull/2162), [@jianjuns])
+- Add support for dumping OVS groups with the "antctl get of" command. ([#1984](https://github.com/antrea-io/antrea/pull/1984), [@jianjuns])
+- Add new "antrea_agent_deny_connection_count" Prometheus metric to keep track of the number of connections denied because of NetworkPolicies; if too many connections are denied within a short window of time, the metric may undercount. ([#2112](https://github.com/antrea-io/antrea/pull/2112), [@zyiou])
+- Generate and check-in clientset code for ClusterGroupMembers and GroupAssociation, to facilitate consumption of these APIs by third-party software. ([#2130](https://github.com/antrea-io/antrea/pull/2130), [@Dyanngg])
+- Package [whereabouts] CNI plugin into the Antrea Linux container image and install the binary on each Node. ([#2185](https://github.com/antrea-io/antrea/pull/2185), [@arunvelayutham])
+- Document requirements for the Node network (how to configure firewalls, security groups, etc.) when running Antrea. ([#2098](https://github.com/antrea-io/antrea/pull/2098), [@luolanzone])
+
+### Changed
+
+- Greatly improve networking performance for Windows Nodes in noEncap mode. [Windows]
+  * Each Agent is responsible for annotating its Node resource with the MAC address of the uplink interface, using the "node.antrea.io/mac-address" annotation ([#2161](https://github.com/antrea-io/antrea/pull/2161), [@tnqn])
+  * Enable [Receive Segment Coalescing (RSC)] in the vSwitch to reduce host CPU utilization and increases throughput for virtual workloads by coalescing multiple TCP segments into fewer, but larger segments. ([#2198](https://github.com/antrea-io/antrea/pull/2198), [@tnqn])
+  * Optimize OVS flows to simplify the forwarding path and increase performance: incoming Pod traffic is forwarded to the destination Pod directly without going through the host network, outgoing Pod traffic is forwarded directly through the uplink when the destination Node is on the same subnet using the MAC address annotation ([#2160](https://github.com/antrea-io/antrea/pull/2160), [@lzhecheng])
```

Comment:
This PR actually adds support for noEncap and hybrid modes for Windows. Probably we should talk about noEncap & hybrid modes support on Windows in Added, before perf improvement?

Reply:
you're right, I split the items accordingly